test/test_new_world_grudge.py

```diff
@@ -0,0 +1,367 @@
+__copyright__ = """
```

Comment:
This file name won't make a great deal of sense once old-world grudge goes away.

Reply:
`test_new_world_grudge.py ` is not intended to remain in the long term. I want to gradually port over tests as more functionally is converted and test them.